memcached-operator/.gitignore

```diff
@@ -78,3 +75,4 @@ tags
 .vscode/*
 .history
 # End of https://www.gitignore.io/api/go,vim,emacs,visualstudiocode
+.idea/
```

Comment:
Is this change necessary? Looks like you just moved the `.idea` entry?

Reply:
I will remove it.